roles/DCOS.agent_windows/vars/main.yml

```diff
@@ -0,0 +1,15 @@
+# DCOS params:
+dcos:
+  version: "1.13"

```

Comment:
this is not a valid DC/OS version. either `1.13.0` or `1.13.1`

Reply:
corrected execution to use -e @/dcos.yml